playgame.py

```diff
@@ -59,17 +65,20 @@ def run_one_game(args):
 def prep_logger(log_dir, verbose, log_stderr, count):
     formatter = logging.Formatter('[%(asctime)s] %(levelname)8s --- %(message)s ' +
                                   '(%(filename)s:%(lineno)s)',datefmt='%Y-%m-%d %H:%M:%S')
+    
     if verbose:
         logger.setLevel(logging.DEBUG)
     else:
         logger.setLevel(logging.INFO)
     if log_dir:
-        fh = RotatingFileHandler('results/{dir}'.format(dir = log_dir), mode = 'w', backupCount = count)
+        ensure_path(log_dir)
+        fh = RotatingFileHandler('{dir}'.format(dir = log_dir), mode = 'w', backupCount = count)
         fh.setLevel(logging.DEBUG)
         logger.addHandler(fh)
         fh.setFormatter(formatter)
     if log_stderr:
-        efh = RotatingFileHandler('results/{dir}'.format(dir = log_stderr), mode = 'w', backupCount = count)
+        ensure_path(log_stderr)
+        efh = RotatingFileHandler('{dir}'.format(dir = log_stderr), mode = 'w', backupCount = count)
```

Comment:
Same as above, we are formatting one string into the same string
```suggestion
        efh = RotatingFileHandler(log_stderr, mode = 'w', backupCount = count)
```